.github/workflows/_binary-test-linux.yml

```diff
@@ -212,7 +212,7 @@ jobs:
           docker-image: ${{ inputs.DOCKER_IMAGE }}
 
       - name: Test Pytorch binary
-        if: ${{ steps.filter.outputs.is-test-matrix-empty == 'False' }}
```

Comment:
This file is likely executed millions of times per day, a better place would be in changing the .yml file so that the test jobs do not even get a runner. E.g., if we disable this test-step, what does the test action do for cuda arm wheel in the yml file?

Reply:
sounds good, let me revert this.